### ISSUE TITLE
Subscription issue

### ISSUE THREAD

**Issue by firstof9** [veista/smartthings#14]
Looks like there maybe an issue:

```
2022-01-25 11:50:21 WARNING (MainThread) [custom_components.smartthings.smartapp] Some device attributes may not receive push updates and there may be subscription creation failures under app '1538476b-1d08-4c29-8434-91ae8b0adb6d' because 55 subscriptions are required but there is a limit of 40 per app
```

Do you think it'd be possible to add separate smartapps for some appliances?

**Comment by veista** [comment]
Yes I guess this is from not removing your unused capabilities. I figured this might happen at one point. I will make a quick fix on next update. Im still figuring out how to proceed with this integration.

**Comment by firstof9** [comment]
Let me know if I can help in any way.

**Comment by veista** [comment]
How many devices you have?

**Comment by veista** [comment]
> Looks like there maybe an issue:
> 
> ```
> 2022-01-25 11:50:21 WARNING (MainThread) [custom_components.smartthings.smartapp] Some device attributes may not receive push updates and there may be subscription creation failures under app '1538476b-1d08-4c29-8434-91ae8b0adb6d' because 55 subscriptions are required but there is a limit of 40 per app
> ```
> 
> Do you think it'd be possible to add separate smartapps for some appliances?

And yes it is possible to do this. Atleast by dividing the the devices in multiple accounts, but this is also part of my doing.

**Comment by firstof9** [comment]
> How many devices you have?

7 devices are listed in SmartThings

2 of which I need to remove from my SmartThings account

**Comment by veista** [comment]
Are they all OCF devices?

**Comment by firstof9** [comment]
Yes all 5 devices are OCF.
